src/dialogs/AccountAuthorizationDialog.vue

```diff
@@ -24,11 +24,11 @@ import { Component, Vue, Prop, Emit } from 'vue-property-decorator'
 
 @Component({})
 export default class AccountAuthorizationDialog extends Vue {
-  // Prop to display the dialog.
-  @Prop() readonly dialog: boolean
+  /** Prop to display the dialog. */
+  @Prop({ default: false }) readonly dialog!: boolean
 
-  // Prop to provide attachment selector.
-  @Prop() readonly attach: string
+  /** Prop to provide attachment selector. */
+  @Prop({ default: '' }) readonly attach!: string
```

Comment:
The default must be an empty string, not null, to avoid warnings in the unit tests.